moviepy/video/fx/headblur.py

```diff
@@ -42,6 +42,7 @@ def fl(gf,t):
         
         orig = im[y1:y2, x1:x2]
         blurred = cv2.blur(orig,(r_blur, r_blur))
+        im=im.copy() # Cannot modify im directly in numpy 1.17
         im[y1:y2, x1:x2] = mask*blurred + (1-mask)*orig
```

Comment:
what about to do this as a particular case, so we don't copy if it isn't strictly needed?
something like 

```suggestion
        try:
            im[y1:y2, x1:x2] = mask*blurred + (1-mask)*orig
        except ValueError:
            # Cannot modify im directly in numpy 1.17
            im= im.copy() 
            im[y1:y2, x1:x2] = mask*blurred + (1-mask)*orig
```

Reply:
Hmm, for me it's a question of which system to optimize for. If, going forward, modifying im directly doesn't work, then that code will in the future always first try the failing code before doing the necessary thing. Your modification optimizes for outdated numpy versions while introducing a penalty for up-to-date versions. Of course, any trick to avoid the copy in current numpy is more than welcome!